lesson2/job1/api.py

```diff
@@ -0,0 +1,59 @@
+import requests
+import os
+
+API_URL = "https://fake-api-vycpfa6oca-uc.a.run.app/"
+
+
+class ApiException(Exception):
```

Comment:
You don't use this exception in the uppermost code (main.py), so it seems this exceptions makes no sense